scss/civicrm/contact/pages/_tag-contacts.scss

```diff
@@ -16,8 +16,52 @@
     vertical-align: middle;
   }
 
+  .jstree-default {
```

Comment:
shouldn't this be part of the context-menu partial?

Reply:
No,
There are 3 parts
1. .`jstree-default` which serves as tree like design (Only in tags)
![image](https://user-images.githubusercontent.com/26058635/35566675-3b85ea9a-05e8-11e8-975b-08f01127f105.png)

2. `.jstree-contextmenu` which is slightly different version of tree (we can say different theme for tree)
![image](https://user-images.githubusercontent.com/26058635/35566759-8b8c290a-05e8-11e8-8627-a6be7d8ec5c6.png)

3. `.jstree-contextmenu` in conjunction with `.vakata-context`
which is actually a context menu and we styled it at `scss/civicrm/common/_context-menu.scss`
![image](https://user-images.githubusercontent.com/26058635/35566866-f2cf1690-05e8-11e8-98aa-fade57b48bcd.png)

Reply:
so we are talking about three different variations of jstree, as far as i understood. Then I think it deserves its own folder with a main file which `@import`s 3 partials for the 3 variations

Reply:
actually, we are only modifying `.jstree-default` which is only for tags. there is not other recurrence of that. and anything else we are no touching. so, we do not have such separate directory. I just gave you variation to explain why the code `.jstree-default` is not belonged to context menu file.

We can do it separate if we are really designing it. but, that's not the case.

Reply:
@mukeshcompucorp you are modifying both `.jstree-default` and `.jstree-contextmenu` with `.vakata-context`, so I think that given both modifications are int he `.jstree-` family, I think it would make the code more organized to group them in a separate directory.

Or at the very least (I think we have folders for pages, not components, correct?) we should put them all in one single partial, where we can collect all these variations without scattering them around

Reply:
yes we manage by page folders so, let me come up with some common partial.